tests/app/tests/renderer-tests.ts

```diff
@@ -1,19 +1,30 @@
 // make sure you import mocha-config before @angular/core
+
 import { assert } from "./test-config";
-import { Component, ElementRef, Renderer2, NgZone, ViewChild } from "@angular/core";
+import { Component, ComponentRef, ElementRef, NgZone, Renderer2, ViewChild } from "@angular/core";
 import { ProxyViewContainer } from "ui/proxy-view-container";
 import { Red } from "color/known-colors";
 import { dumpView } from "./test-utils";
 import { TestApp } from "./test-app";
-import { isIOS } from "tns-core-modules/platform";
-import { View, fontInternalProperty, backgroundInternalProperty} from "tns-core-modules/ui/core/view"
 import { LayoutBase } from "tns-core-modules/ui/layouts/layout-base";
 import { StackLayout } from "tns-core-modules/ui/layouts/stack-layout";
 import { ContentView } from "tns-core-modules/ui/content-view";
+import { NgView, registerElement } from "nativescript-angular/element-registry";
 import { Button } from "tns-core-modules/ui/button";
-import { NgView } from "nativescript-angular/element-registry";
-import { registerElement } from "nativescript-angular/element-registry";
-// import * as view from "tns-core-modules/ui/core/view";
+import * as view from "tns-core-modules/ui/core/view";
+import { isIOS } from "tns-core-modules/platform";
+import { View, fontInternalProperty, backgroundInternalProperty } from "tns-core-modules/ui/core/view"
+import { nsTestBedAfterEach, nsTestBedBeforeEach, nsTestBedRender } from "nativescript-angular/testing";
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { Observable } from "rxjs/Observable";
+import { ReplaySubject } from "rxjs/ReplaySubject";
```

Comment:
Import `Observable` and `ReplaySubject` directly form `rxjs` -  it is rxjs@6 way :)
```
import { Observable, ReplaySubject } from "rxjs";
```

Reply:
already indoor PR ;)